docs/pages/guides/monorepos.mdx

```diff
@@ -153,6 +161,10 @@ We have to tell Metro to look in these two folders. The order is important here
 
 <Collapsible summary="3. Why do we need to disable the hierarchical lookup?">
 
+<Callout type="warning">
+  Starting from Expo SDK 50, the hierarchical lookup should no longer be disabled by default. Both React Native and Expo has improved general monorepo support, and disabling this option can cause other issues. Only enable this option if you know what you are doing.
```

Comment:
```suggestion
  Starting from Expo SDK 50, the hierarchical lookup should no longer be disabled by default. Both React Native and Expo have improved general monorepo support, and disabling this option can cause other issues. Only enable this option if you know what you are doing.
```

It sounds a bit conflicting. So, if I am using SDK 50, I should no longer disable it. However, the last line says only enable it when I know what I'm doing. Can we either separate that and elaborate on it more?

Reply:
Yeah, but it's "should no longer be disabled by default". If you have reasons to, you can enable it. The "Only enable this option if you know what you are doing." could be removed I guess

Reply:
```suggestion
  Starting from Expo SDK 50, the hierarchical lookup should no longer be disabled by default. Both React Native and Expo have improved general monorepo support, and disabling this option can cause other issues.
```